src/tools/kata-ctl/src/arch/x86_64/mod.rs

```diff
@@ -141,6 +168,120 @@ mod arch_specific {
 
         Ok(GuestProtection::NoProtection)
     }
+
+    fn running_on_vmm() -> Result<bool> {
+        match check::get_single_cpu_info(check::PROC_CPUINFO, CPUINFO_DELIMITER) {
+            Ok(cpu_info) => {
+                // check if the 'hypervisor' flag exist in the cpu features
+                let missing_hypervisor_flag = check::check_cpu_attribs(&cpu_info, VMM_FLAGS)?;
+
+                if missing_hypervisor_flag.is_empty() {
+                    return Ok(true);
+                }
+            }
+            Err(e) => {
+                return Err(anyhow!(
+                    "Unable to determine if the OS is running on a VM because this error: {}: {}",
```

Comment:
```suggestion
                  "Unable to determine if the OS is running on a VM: {}: {}",
```

("because this error" is not grammatically correct, I believe).